### ISSUE TITLE
Wii U Pro Controller gets disconnected

### ISSUE THREAD

**Issue by sune26** [cemu-project/Cemu#1163]
### Current Behavior

As soon as I start CEMU on windows the controller disconnects, it did not do this on the previous version I was on. This is happening on 2.0-74.

### Expected Behavior

For it to not disconnect as soon as I start CEMU.

### Steps to Reproduce

Connect Wii U Pro controller to windows, start cemu.

**Comment by Aspenpaw** [comment]
I'm having the same issue, with a Switch Pro controller. It only happens when using the SDL API. It does not happen with my Xbox Series X controller, (regardless of API), I have eliminated Steam as an issue (I closed right out of it while testing). I would call this issue fairly important as I cannot complete the Myahm Agana shrine without it (I need a gyro and cemu-no-gyro is terrible) :(

**Comment by thanone2** [comment]
I have the same issue too

**Comment by Hydraxonn** [comment]
I also have this issue on the latest version. I believe it was around 1.8 that this still worked. Would appreciate this being looked into!

**Comment by Hydraxonn** [comment]
OK, just did some more digging. It broke between 2.0-59 and 2.0-60. I suspect it's this commit: https://github.com/cemu-project/Cemu/commit/18490830738c61b1b35fa11a9207bcf3fc4edd3e

 Gonna try removing this system in a custom fork and see if it fixes the Wii U Pro Controller compatibility.

**Comment by Hydraxonn** [comment]
Yes, it simply seems that setting ENABLE_HIDAPI to false on line 94 of CMakeLists.txt fixes this issue. https://github.com/cemu-project/Cemu/commit/18490830738c61b1b35fa11a9207bcf3fc4edd3e#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR94

**Comment by capitalistspz** [comment]
You can use [HidHide](https://docs.nefarius.at/projects/HidHide/) on the controller so that Cemu only sees the emulated Xbox controller.
This will stop the controller from being disconnected.

![image](https://github.com/user-attachments/assets/4e4e06dc-bf4f-4a9d-8456-709df876acbd)

**Comment by Ak8orious** [comment]
> Yes, it simply seems that setting ENABLE_HIDAPI to OFF on line 94 of CMakeLists.txt fixes this issue. [1849083#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR94](https://github.com/cemu-project/Cemu/commit/18490830738c61b1b35fa11a9207bcf3fc4edd3e#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR94)

Where do i find that txt file on cemu 2.4?

**Comment by djey47** [comment]
> > Yes, it simply seems that setting ENABLE_HIDAPI to OFF on line 94 of CMakeLists.txt fixes this issue. [1849083#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR94](https://github.com/cemu-project/Cemu/commit/18490830738c61b1b35fa11a9207bcf3fc4edd3e#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR94)
> 
> Where do i find that txt file on cemu 2.4?

That's source. One then have to rebuild a fixed release afterwards...

But noone seems to consider this issue anymore; I'm using hidhide as advised, it's not the best solution but it works at least.

**Comment by Zakarychu** [comment]
Will never be fixed? Version 2.6 as today same behavior.

**Comment by capitalistspz** [comment]
> Will never be fixed? Version 2.6 as today same behavior.

The only way to fix it is to prevent Cemu from opening connected Wii remotes (Wii U Pro Controller is indistinguishable from a Wiimote until opened). 

The best solution would be to make opening require user interaction
